### PR TITLE
Single file support

### PR DESCRIPTION
**Pull request by smiklos**
Remove stripping of [] from fileupload form fields. 
Added Single file support to fileuploader and FileInput/UdashForm

### CODE REVIEW

File: bootstrap/.js/src/main/scala/io/udash/bootstrap/form/UdashForm.scala [hunk 1]
```diff
@@ -203,16 +203,34 @@ object UdashForm {
...
     * @param name                Name of the input. This value will be assigned to the `name` attribute of the input.
-    * @param acceptMultipleFiles If true, input will accept multiple files.
-    * @param selectedFiles       Property which will be synchronised with the input content.
+    * @param selectedFile        Property which will be synchronised with the input content.
     * @param inputModifiers      Modifiers applied directly to the `input` element.
     */
   def fileInput(inputId: ComponentId = ComponentId.newId(), validation: Option[Modifier] = None)
+    (labelContent: Modifier*)
+    (name: String,
+      selectedFile: Property[File], inputModifiers: Modifier*): Modifier =
```
Comment (ddworak) [inline bootstrap/.js/src/main/scala/io/udash/bootstrap/form/UdashForm.scala]: nitpick: you can remove the line break, fits a single line with `name` easily

Comment (ddworak) [inline bootstrap/.js/src/main/scala/io/udash/bootstrap/form/UdashForm.scala]: nitpick: you can remove the line break, fits a single line with name easily

File: bootstrap/.js/src/main/scala/io/udash/bootstrap/form/UdashForm.scala [hunk 2]
```diff
@@ -203,16 +203,34 @@ object UdashForm {
...
+    * @param inputModifiers      Modifiers applied directly to the `input` element.
+    */
+  def multipleFileInput(inputId: ComponentId = ComponentId.newId(), validation: Option[Modifier] = None)
                (labelContent: Modifier*)
-               (name: String, acceptMultipleFiles: ReadableProperty[Boolean],
+               (name: String,
                 selectedFiles: SeqProperty[File], inputModifiers: Modifier*): Modifier =
     inputGroup(inputId, validation)(labelContent)(
-      FileInput(selectedFiles, acceptMultipleFiles)(name, id := inputId, inputModifiers).render
+      FileInput(selectedFiles, Property(true))(name, id := inputId, inputModifiers).render
```
Comment (ddworak) [inline bootstrap/.js/src/main/scala/io/udash/bootstrap/form/UdashForm.scala]: Use `true.toProperty` which creates an optimized immutable property

**Comment by smiklos** [comment]
@ddworak made the changes

**Comment by smiklos** [comment]
Any ETA for the new version?

**Comment by ddworak** [comment]
Releasing RC8 right now

**Comment by smiklos** [comment]
Looks like git tagging didn't work for the last release

**Pull request merged**